keyman_trust_linux.go

```diff
@@ -4,71 +4,90 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"os/user"
+	"path/filepath"
 )
 
-func DeleteTrustedRootByName(commonName string, prompt string) error {
-	nssdb, err := getUserNssdb()
-	if err != nil {
-		return err
+var (
+	nssDBs = []string{
+		filepath.Join(os.Getenv("HOME"), ".pki/nssdb"),
```

Comment:
why not use os/user here just to be safe?

Reply:
@oxtoacart 
I don't mind, just curious, what case are we protecting from? :)
HOME is always defined AFAIK

Reply:
Ahh cool, looks like POSIX actually requires it be defined. I thought it was just a shell thing, but it's not. So, I'm cool with it.